pyproject.toml

```diff
@@ -1,3 +1,14 @@
 [build-system]
-requires = ["setuptools", "wheel", "urllib3<1.26", "conan>=1.22.2", "scikit-build", 
-  "cmake!=3.17.1,!=3.17.0", "ninja", "pybind11>2.4", "Cython>0.27.1", "numpy>1.16.3"]
+requires = [
+  "setuptools",
+  "wheel",
+  "conan>=1.31.2",
```

Comment:
I think the PR that updates minimum `conan` to 1.31.2 and gets rid of `urllib3` didn't make it to `aer 0.7`. So maybe we should keep  `"urllib3<1.26", "conan>=1.22.2"` here.

Reply:
Yep, good call. This must of slipped in when I rebased the PR from master to stable/0.7. Fixed in: https://github.com/Qiskit/qiskit-aer/pull/1131/commits/5ce378f2b4b3f3bfb6ffe0907f11ca6f55cc377a